connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectorValidationIntegrationTest.java

```diff
@@ -520,17 +543,6 @@ public byte[] fromConnectHeader(String topic, String headerKey, Schema schema, O
     public abstract static class AbstractTestConverter extends TestConverter {
     }
 
-    public static class TestConverterWithPrivateConstructor extends TestConverter {
-        private TestConverterWithPrivateConstructor() {
-        }
-    }
-
-    public static class TestConverterWithConstructorThatThrowsException extends TestConverter {
-        public TestConverterWithConstructorThatThrowsException() {
-            throw new ConnectException("whoops");
-        }
-    }
-
     public static class TestConverterWithSinglePropertyConfigDef extends TestConverter {
```

Comment:
This and the other plugin classes below it seem fine to leave in for now, since (being valid plugins that can be successfully instantiated via a public, no-args constructor) they don't generate any abnormal log messages on worker startup.